internal/pkg/action/operator_install.go

```diff
@@ -51,7 +51,10 @@ func (i *OperatorInstall) BindFlags(fs *pflag.FlagSet) {
 	fs.BoolVarP(&i.CreateOperatorGroup, "create-operator-group", "C", false, "create operator group if necessary")
 
 	fs.VarP(&i.InstallMode, "install-mode", "i", "install mode")
-	fs.MarkHidden("install-mode")
+	err := fs.MarkHidden("install-mode")
+	if err != nil {
+		log.Print(`requested flag "install-mode" missing`)
```

Comment:
`panic` here maybe? This error would only happen if we screw up and change the name of the flag without changing the call to `MarkHidden`.